examples/seismic/tutorials/02_rtm.ipynb

```diff
@@ -470,7 +470,7 @@
    ],
    "source": [
     "# Run imaging loop over shots\n",
-    "from devito import Function, clear_cache\n",
+    "from devito import Function\n",
```

Comment:
So nitpicking, but currently `u0` is still created once at the beginning and passed to the operator for each shot so memory overhead is still tiny without clear_cache the way the loop is implemented. Would be better to remove `u0=..., u0.data.fill` and actually see if creating a new `u` at each shots doesn't blow up memory. Same for FWI.